Support/Scripts/tidy.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3.7
+
+import argparse
+import json
+import subprocess
+import os
+import sys
+
+# This script assumes that there is a compile_commands.json in the current
+# directory. If there is not, you must pass 
+# `--ccdb path/to/compile_commands.json`.  
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--ccdb",
+        default="./compile_commands.json",
```

Comment:
I think if you remove the `./` you keep the same behavior but make your code more portable (Windows...).